internal/pkg/skuba/addons/psp.go

```diff
@@ -31,6 +31,66 @@ const (
 	pspManifest = `---
 apiVersion: policy/v1beta1
 kind: PodSecurityPolicy
+metadata:
+  name: suse.caasp.psp.cilium
+spec:
+  privileged: true
+  allowPrivilegeEscalation: true
+  allowedCapabilities:
+    - NET_ADMIN
+    - SYS_MODULE
+  volumes:
+    - 'configMap'
+    - 'emptyDir'
+    - 'projected'
+    - 'secret'
+    - 'downwardAPI'
+    - 'hostPath'
+  hostNetwork: true
+  hostIPC: false
+  hostPID: false
+  runAsUser:
+    rule: 'RunAsAny'
+  seLinux:
+    rule: RunAsAny
+  supplementalGroups:
+    rule: 'MustRunAs'
+    ranges:
+      - min: 1
+        max: 65535
+  fsGroup:
+    rule: 'MustRunAs'
+    ranges:
+      - min: 1
+        max: 65535
+  readOnlyRootFilesystem: false
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: suse:caasp:psp:cilium
+rules:
+- apiGroups: ['policy']
+  resources: ['podsecuritypolicies']
+  verbs:     ['use']
+  resourceNames:
+  - cilium-psp
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: suse:caasp:psp:cilium-binding
```

Comment:
In cilium.go, you are defining another ClusterRoleBinding named cilium, shouldn't these be defined in the same file to organize things better?

Reply:
Yes, I agree for that we have to use `suse.caasp.psp.privileged`

Reply:
Thanks for pointing that out! I switched to `suse.caasp.psp.privileged`.